### PR TITLE
add to_text_encoding_none test

### PR DESCRIPTION
**Pull request by guilhermeleobas**
As title.

### CODE REVIEW

File: rbc/tests/heavydb/test_text_encoding_dict.py [hunk 1]
```diff
@@ -114,6 +115,28 @@ def fn_copy(mgr, t):
...
+        @heavydb('TextEncodingNone(RowFunctionManager, TextEncodingDict)', devices=['cpu'])
+        def to_text_encoding_none_1(mgr, t):
+            db_id = mgr.getDictDbId('to_text_encoding_none_1', 0)
+            dict_id = mgr.getDictId('to_text_encoding_none_1', 0)
+            str = mgr.getString(db_id, dict_id, t)
+            n = len(str)
+            r = TextEncodingNone(n)
+            for i in range(n):
+                r[i] = str[i]
+            return r
```
Comment (pearu) [inline rbc/tests/heavydb/test_text_encoding_dict.py:127]: Why there is two UDFs with identical implementations?
Did you mean:
```suggestion
            return str
```
?

**Review by pearu — CHANGES_REQUESTED** [review]
Thanks for the PR, @guilhermeleobas!

File: rbc/tests/heavydb/test_text_encoding_dict.py (hunk 1 above)
Comment (guilhermeleobas) [reply rbc/tests/heavydb/test_text_encoding_dict.py]: Oops, my bad. Fixed

**Review by pearu — COMMENTED** [review]
LGTM!

**Pull request merged**